apps/aehttp/priv/oas3.yaml

```diff
@@ -535,6 +535,27 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+  /hyperchain/contract-pubkeys:
```

Comment:
Why the obsession with the term 'pubkey' here? What else would the endpoint return (the bytecode?) I'd name the endpoint `/hyperchain/contracts`?

Reply:
True. Will resolve these two then. Look forward to new CCI run and re-approval...